cli/src/semgrep/dependency_aware_rule.py

```diff
@@ -109,7 +109,7 @@ def generate_unreachable_sca_findings(
                         # Output_from_core.atd so we can reuse out.MatchExtra
```

Comment:
Should the CLI also print something near these "Pro-only" findings ?

Reply:
I'd like to do that, but I hear osemgrep might fully port output to OCaml soon so if I wait then I don't have to duplicate work

Reply:
Have you checked if it a lot of work ? There is already code somewhere to add those ">>>" marks to high severity findings. It may just be another `if` adding a "ruby" or something.

Reply:
Even if you don't do it in this PR I think it's worth pinging "ZZ", I presume Growth is interested in doing it? And he seems to be quite familiar already with pysemgrep's code.

Reply:
Either way I'll merge this first!